addons/account/static/src/scss/account_journal_dashboard.scss

```diff
@@ -33,6 +33,10 @@
         }
     }
 
+    .o_favorite{
+        padding-top: 0;
+    }
```

Comment:
I don't understand why we have to do that here since we don't have `o_account_kanban` class in kanban view of project.task...

Reply:
I think this changes is useless and just the changes in web will do the job. 🙂

Reply:
Hello @xavierbol 
In this PR-https://github.com/odoo/odoo/pull/124416/files they have remove the
  padding as they have alignment issue in accounting dashboard but due to that
  fix it is breaking project kanban card view favorite icon
  
  https://tinyurl.com/2bpvehmz  so for this alignment issue we have added padding 0 so that it doesn;t break anything